### PR TITLE
Allow option for antch to set HTTP2ForceUpgrade transport param 

### PR DESCRIPTION
**Pull request by vivekraghunathan**
In follow up, I might propose to change crawler interface to allow a transport to be injected (or a http client). For now, this felt the minimal change to fix the bug where using a DialContext for HTTP Transport to use HTTP/2

### CODE REVIEW

File: crawler.go [hunk 1]
```diff
@@ -47,6 +47,9 @@ type Crawler struct {
 	// UserAgent specifies the user-agent for the remote server.
 	UserAgent string
 
+	// ForceAttemptHTTP2 specifies the corresponding field in the underlying HTTP transport.
+	ForceAttemptHTTP2 bool
```
Comment (asimshankar) [inline crawler.go]: Instead of making this an option, I'd just remove the option and force it to `true` in line 202.
It seems that doing so is better behavior now (and consistent with `http.DefaultTransport`?)

File: robotstxt_test.go [hunk 2]
```diff
@@ -31,7 +31,7 @@ func TestRobotstxtHandler(t *testing.T) {
 	}{
 		{"/", "", true},
 		{"/shopping/", "", true},
-		{"account", "", false},
```
Comment (asimshankar) [inline robotstxt_test.go:34]: Why this change?

File: go.mod [hunk 3]
```diff
@@ -0,0 +1,15 @@
+module github.com/neevaco/antch
```
Comment (asimshankar) [inline go.mod]: Optional: Do you want to split out the `go.mod` changes into a separate PR, or at least attempt to upstream that separately?

**Review by asimshankar — APPROVED** [review]
Let's make these changes to the `neeva` branch instead of `master` branch so that `master` reflects upstream and `neeva` reflects all our changes?

File: crawler.go (hunk 1 above)
Comment (vivekraghunathan) [reply crawler.go]: done

File: robotstxt_test.go (hunk 2 above)
Comment (vivekraghunathan) [reply robotstxt_test.go:34]: the unit tests didn't pass.

File: go.mod (hunk 3 above)
Comment (vivekraghunathan) [reply go.mod]: done .. removed from this PR
